Source/WebCore/style/AttributeChangeInvalidation.h

```diff
@@ -33,22 +33,22 @@ namespace Style {
 
 class AttributeChangeInvalidation {
 public:
-    AttributeChangeInvalidation(Element&, const QualifiedName&, const AtomString& oldValue, const AtomString& newValue);
+    AttributeChangeInvalidation(Ref<Element>&&, const QualifiedName&, const AtomString& oldValue, const AtomString& newValue);
     ~AttributeChangeInvalidation();
 
 private:
     void invalidateStyle(const QualifiedName&, const AtomString& oldValue, const AtomString& newValue);
     void invalidateStyleWithRuleSets();
 
     const bool m_isEnabled;
-    Element& m_element;
+    Ref<Element> m_element;
```

Comment:
This is a little risky performance-wise as I suspect this may be hot code. I have started performance A/B testing to validate this PR.